src/components/WalletDropdown/TransactionMenu.tsx

```diff
@@ -0,0 +1,11 @@
+import { Trans } from '@lingui/macro'
```

Comment:
this is a placeholder for now, will open another pr with all transaction logic later